test/censored.jl

```diff
@@ -0,0 +1,368 @@
+# Testing censored distributions
+
+module TestCensored
+
+using Distributions, Test
+using Distributions: Censored
+
+function _as_mixture(d::Censored)
+    d0 = d.uncensored
+    dtrunc = if d0 isa DiscreteUniform || d0 isa Poisson
+        truncated(
+            d0,
+            d.lower === missing ? -Inf : floor(d.lower) + 1,
+            d.upper === missing ? Inf : ceil(d.upper) - 1,
+        )
+    elseif d0 isa ContinuousDistribution
+        truncated(
+            d0,
+            d.lower === missing ? -Inf : d.lower + eps(float(d.lower)),
+            d.upper === missing ? Inf : d.upper - eps(float(d.upper)),
```

Comment:
Would it be sufficient to use
```suggestion
            d.lower === missing ? -Inf : nextfloat(float(d.lower)),
            d.upper === missing ? Inf : prevfloat(float(d.upper)),
```
?